src/main.rs

```diff
@@ -20,44 +20,51 @@ fn main() {
 
     let filepath = &args[1];
 
-    javap(filepath);
+    let javap_result = javap(filepath, false);
+    println!("{}", javap_result);
 }
 
-fn javap(filepath: &str) {
+fn javap(filepath: &str, print_code: bool) -> String {
```

Comment:
In the future we should probably change this to write to an output stream rather than building it up as a single string. That way we could maintain the testability by passing in a writer that the test could later convert to a string, while using stdout for the executable.

But this is good for now.

Reply:
I opened #7 for this.

Reply:
yeah, i guess that would be a little more idiomatic. thanks!